README.md

```diff
@@ -8,8 +8,8 @@
 
 [crates-badge]: https://img.shields.io/crates/v/capsule.svg
 [crates-url]: https://crates.io/crates/capsule
-[docs-badge]: https://docs.rs/capsule/badge.svg?version=0.1.0
-[docs-url]: https://docs.rs/capsule/0.1.0/capsule
+[docs-badge]: https://docs.rs/capsule/badge.svg?version=0.1.2
+[docs-url]: https://docs.rs/capsule/0.1.2/capsule
```

Comment:
`https://docs.rs/capsule/badge.svg` and `https://docs.rs/capsule/` automatically links you to the latest version, I think we can go with that.